### PR TITLE
New Term: Aggressive Scan

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

New Nmap entry

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by KTom101 — CHANGES_REQUESTED** [review]
@SSwiniarski I've added a few suggestions for this entry, thanks!

**Comment by SSwiniarski** [comment]
@KTom101, I implemented your changes.

**Review by KTom101 — APPROVED** [review]
@SSwiniarski Looks good for the next review 👍

File: content/cybersecurity/concepts/nmap/terms/aggressive-scan/aggressive-scan.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+Title: 'Aggressive Scan'
+Description: 'An aggressive scan provides far better information than a standard scan, but is more likely to be detected.'
```
Comment (Dusch4593) [inline content/cybersecurity/concepts/nmap/terms/aggressive-scan/aggressive-scan.md:3]:
```suggestion
Description: 'Provides far better information than a standard scan, but is more likely to be detected.'
```

File: content/cybersecurity/concepts/nmap/terms/aggressive-scan/aggressive-scan.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+nmap -A <target>
+```
+
+The above conducts an aggressive scan on `<target>`.
+
+## Example
+
+The following example runs an aggressive scan on the site `scanme.nmap.org`:
+
+````shell
```
Comment (Dusch4593) [inline content/cybersecurity/concepts/nmap/terms/aggressive-scan/aggressive-scan.md:30]:
```suggestion
```shell
```

File: content/cybersecurity/concepts/nmap/terms/aggressive-scan/aggressive-scan.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+nmap -A scanme.nmap.org
+
```
Comment (Dusch4593) [inline content/cybersecurity/concepts/nmap/terms/aggressive-scan/aggressive-scan.md:32]: We're missing closing backticks here.

File: content/cybersecurity/concepts/nmap/terms/aggressive-scan/aggressive-scan.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+> **Note:** The site `scanme.nmap.org` is a site explicitly set up for testing Nmap. Only perform scans on domains you have explicit permission to scan.
```
Comment (Dusch4593) [inline content/cybersecurity/concepts/nmap/terms/aggressive-scan/aggressive-scan.md:33]:
```suggestion
> **Note:** This is a site explicitly set up for testing Nmap. Scans should only be performed on domains with explicit permissions given to the user.
```

File: content/cybersecurity/concepts/nmap/terms/aggressive-scan/aggressive-scan.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+```pseudo
+nmap -A <target>
+```
+
+The above conducts an aggressive scan on `<target>`.
+
+## Example
+
+The following example runs an aggressive scan on the site `scanme.nmap.org`:
```
Comment (Dusch4593) [inline content/cybersecurity/concepts/nmap/terms/aggressive-scan/aggressive-scan.md:28]:
```suggestion
The following example runs an aggressive scan on the site [scanme.nmap.org](http://scanme.nmap.org/):
```

File: content/cybersecurity/concepts/nmap/terms/aggressive-scan/aggressive-scan.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+Tags:
+  - 'Networking'
+  - 'Cybersecurity'
+  - 'Ethical Hacking'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/scan-systems-with-nmap'
+---
+
+An **aggressive scan** provides far better information than a [regular scan](https://www.codecademy.com/resources/docs/cybersecurity/nmap/nmap-scan), but is more likely to be detected. It is performed by using the `-A` option. The aggressive scan enables OS detection, version detection, script scanning, and traceroute. It sends out more probes than a regular scan, so it is more likely to be detected during a security audit.
```
Comment (Dusch4593) [inline content/cybersecurity/concepts/nmap/terms/aggressive-scan/aggressive-scan.md:16]:
```suggestion
An **aggressive scan** provides far better information than a [regular scan](https://www.codecademy.com/resources/docs/cybersecurity/nmap/nmap-scan), but is more likely to be detected. It is performed by using the `-A` option and enables the following:

- OS detection (`-o`)
- Version detection (`-sV`)
- Script scanning (`-sC`)
- Traceroute (`--traceroute`)

Aggressive scans send out more probes than a regular scan, and are more likely to be detected during a security audit.
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Finished with my review and left some comments/suggestions.

**Review by Dusch4593 — APPROVED** [review]
@SSwiniarski LGTM! 👍🏻

**Pull request merged**